installing/installing_openstack/installing-openstack-user.adoc

```diff
@@ -0,0 +1,81 @@
+[id="installing-openstack-user"]
+= Installing a cluster on OpenStack on your own infrastructure
+include::modules/common-attributes.adoc[]
+:context: installing-openstack-user
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+{rh-openstack-first} that runs on infrastructure that you provide.
+
+Using your own infrastructure allows you to integrate your cluster with existing infrastructure and modifications. The process requires more labor on your part than installation-program-provisioned installations, however, as you must create all {rh-openstack} resources, like Nova servers, Neutron ports, and security groups.
+
+.Prerequisites
+
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* Have access to an {rh-openstack} administrator's account
+
+* On the target installation environment, have a single directory in which you can keep the files you create during the installation process
+
+* On the target installation environment, install:
+** Python
+** The following Python modules:
+*** ansible version 2.9.2 or compatible
+*** openstacksdk version 0.39.0 or compatible
+*** openstackclient version 4.0.0 or compatible
+*** netaddr version 0.7.19 or compatible
+
+[TIP]
+====
+To install those modules quickly, copy these lines into a `requirements.txt` file and, in a command-line prompt, enter `pip -r requirements.txt`:
```

Comment:
That would be `pip install -r requirements.txt` (you're missing the command "install")

Reply:
Whoops. Thanks!

Reply:
or after ‘python -m venv  venv && source venv/bin/activate’